src/lib/button-toggle/button-toggle.html

```diff
@@ -1,22 +1,18 @@
-<label [attr.for]="inputId" class="mat-button-toggle-label" #label>
-  <input #input class="mat-button-toggle-input cdk-visually-hidden"
-         [type]="_type"
-         [id]="inputId"
-         [checked]="checked"
-         [disabled]="disabled || null"
-         [attr.name]="name"
-         [attr.aria-label]="ariaLabel"
-         [attr.aria-labelledby]="ariaLabelledby"
-         (change)="_onInputChange($event)"
-         (click)="_onInputClick($event)">
-
+<button #button class="mat-button-toggle-button"
```

Comment:
This needs `type="button"`, otherwise it'll submit any parent form elements when it's toggled.